src/encoded/static/components/app.js

```diff
@@ -209,11 +209,40 @@ export default class App extends React.PureComponent {
             store.dispatch({ type: { href: windowHref } });
         }
 
+        const analyticsOptions = _.extend({
+            "itemToProductTransform"  : function(item){
+                const {
+                    "@id": itemID,
+                    uuid: itemUUID,
+                    "@type" : itemType,
+                    display_title,
+                    title,
+                    submission_centers: [ { display_title: submissionCenterTitle } ] = [{}],
+                    file_sets: [ { assay: { display_title: assayTitle } = {} } = {} ] = [{}],
+                    data_category: [ file_type ] = []
+                } = item;
+        
+                const categories = Array.isArray(itemType) ? itemType.slice().reverse().slice(1) : [];
+                const prodItem = {
+                    'item_id': itemID || itemUUID,
+                    'item_name': display_title || title || null,
+                    'item_category': categories.length >= 1 ? categories[0] : "Unknown",
+                    'item_category2': categories.length >= 2 ? categories[1] : "Unknown",
+                    'item_category3': categories.length >= 3 ? categories[2] : "Unknown",
+                    'item_brand': submissionCenterTitle || null,
+                    'experiment_type': assayTitle || null,
```

Comment:
Out of curiosity, why is this still `experiment_type`?

Reply:
@Bianca-Morris  we are still using analytics utils in shared-portal-components that was previously implemented for 4DN.